Resources/Prototypes/_RMC14/Roles/Jobs/Fun/rmc_admin_ghost.yml

```
@@ -11,9 +12,11 @@
   id: RMCMobAghostGear
   equipment:
     back: ClothingBackpackSatchelHolding
-    id: CMIDCardHighCommand # RMC-14: Changed from AdminPDA
+    id: RMCAdminPDA
   storage:
     back:
-    # RMC-14: Replaced loadout
-    - CMHealthAnalyzer
     - RMCAdminAccessConfigurator
```

put the base default analyzer here as well for deeper damage debugging as well
Not sure if this is the right id
```suggestion
    - RMCAdminAccessConfigurator
    - HealthAnalyzer
```